consistency/tool/clustering/word_extraction_test.json

```diff
@@ -0,0 +1,49 @@
+{
+
+  "test1": [ 
+	
+   { 
+        "name":"artifact1", 
+        "schemas":     [{"word": "ab", "count":2}, {"word": "bc", "count":1},{"word": "cd", "count":1} ],
+        "properties" : [{"word": "ab", "count":2}, {"word": "bc", "count":1},{"word": "cd", "count":1} ],
+        "operations" : [{"word": "ab", "count":2}, {"word": "bc", "count":1},{"word": "cd", "count":1} ],
+        "parameters":  [{"word": "ab", "count":2}, {"word": "bc", "count":1},{"word": "cd", "count":1} ]
+     }, 
+
+     { 
+      "name":"artifact2", 
+      "schemas":     [{"word": "ab", "count":2}, {"word": "bc", "count":1},{"word": "cd", "count":1} ],
+      "properties" : [{"word": "ab", "count":2}, {"word": "bc", "count":1},{"word": "cd", "count":1} ],
+      "operations" : [{"word": "ab", "count":2}, {"word": "bc", "count":1},{"word": "cd", "count":1} ],
+      "parameters":  [{"word": "ab", "count":2}, {"word": "bc", "count":1},{"word": "cd", "count":1} ]
+   }
+    
+  ], 
+
+  "test2": [ 
```

Comment:
Can we take out this test case since it is similar to test1. We can add more cases later.